design-docs/009-tigris-support.md

```diff
@@ -0,0 +1,50 @@
+# Tigris support
+
+[Tigris](https://www.tigrisdata.com/) is a globally distributed S3-compatible object store. Writes and reads get dispatched
+automatically to the geographically closest available region, and then the data gets automatically
+replicated to other regions.
+
+## Why is support needed
+
+Tigris doesn't offer read-after-create or read-after-update consistency by default. Data may be written to a region
+and read from another, where the data hasn't replicated yet.
+
+This means basic Icechunk guarantees cannot be hold. For example, something as simple as creating a repo can fails
+a high percentage of the time in certain geographic configurations.
+
+Tigris has built for us the ability to recover consistency by passing an `X-Tigris-Regions` header in the request. If
+writes and reads are done with the same region in the header, consistency is recovered. We also need `Cache-Control:no-cache`
+in the get requests. Of course, this comes at the price of performance, because reads no longer happens from the closest region anywhere in the world.
+
+An alternative would be to set the bucket as region-restricted. In that case all writes and reads go to the single region,
+but Icechunk cannot control the bucket configuration.
```

Comment:
What does "Icechunk cannot control the bucket configuration" mean?

Reply:
i'll clarify